python/graphstorm/eval/evaluator.py

```diff
@@ -466,6 +843,9 @@ def compute_score(self, pred, labels):
 class GSgnnAccEvaluator(GSgnnInstanceEvaluator):
```

Comment:
Do you want to remove this function in this PR?

Reply:
No. I would like to remove all deprecated tests after all Evaluators are replaced with new ones.